tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -159,8 +183,52 @@ public function testAttributeKeyCharacterLimitCausesException()
      * @param int $stringLength The string length to generate.
      * @return string The multi byte character string.
      */
-    public function generateMultiByteCharacterString($stringLength)
+    public static function generateMultiByteCharacterString($stringLength)
```

Comment:
Why using a static helper here?

Reply:
See [this comment](https://github.com/findologic/libflexport/pull/98#discussion_r267884157).